src/tools/cephfs/JournalTool.cc

```diff
@@ -367,6 +369,15 @@ int JournalTool::main_event(std::vector<const char*> &argv)
     return r;
   }
 
+  // Check if need to repair
+  for (std::vector<const char*>::iterator iter = argv.begin(); iter != argv.end(); iter++) {
```

Comment:
Please use:

`for (const auto& s : argv) {`

Reply:
Also, why is this option code not in `JournalTool::main`?

Reply:
the option only be used in JournalTool::main_event, so i think we don't need put it in JournalTool::main.Secondly, we need pass the repaire option to JournalTool::main_event or define a member variable of JournalTool if we code this option in JournalTool::main.